src/oas3/transformers/callbacks.ts

```diff
@@ -0,0 +1,28 @@
+import { IHttpCallbackOperation } from '@stoplight/types';
+import { CallbacksObject } from 'openapi3-ts';
+import { transformOas3Operation } from '../operation';
+
+export function translateToCallbacks(callbacks?: CallbacksObject): IHttpCallbackOperation[] | undefined {
```

Comment:
I changed the input arg to required. I vote on keeping function return value nullable. The reason is that it is most convenient way of not adding callbacks field to all `IHttpOperation` instances. I don't want to check for `Object.entries(operation).length` at `transformOas3Operation` level because it won look nice.

Reply:
Let's return `Option< IHttpCallbackOperation[]>` :trollface: 

Oh no, this is not Prism.

Reply:
`Option<NonEmptyArray<IHttpCallbackOperation[]>>` 🚎